docs/javascript/v2/how--to-guides/debugging/faq.mdx

```diff
@@ -168,3 +168,41 @@ You can get the HLS URL in several ways based on whether you use 100ms client SD
 #### I get type errors related to WebRTC Stats, for example, 'Cannot find name RTCInboundRtpStreamStats'.
 
 The minimum supported version of TypeScript for the SDK is '4.4'. You could update to any version above 4.4 or if you are on an old version of TypeScript and cannot do a major upgrade you can set `skipLibCheck: true` in your tsconfig file.
+
+#### Why is my screenshare not working on mobile browsers?
+
+The screenshare API(getDisplayMedia) is not yet supported on mobile browsers. Refer [this](https://caniuse.com/?search=getDisplayMedia) for supported browsers.
+
+
+#### Can I use the sdk to join multiple rooms at the same time?
```

Comment:
can this be a standalone page? FAQs don't help in discoverability - too long a list

Reply:
should this be part of join or a separate page itself?

Reply:
as discussed will be added with a demo.